### PR TITLE
ENH: full raster on val

### PR DESCRIPTION
**Pull request by eigenfoo**
1. Removes a lot of unneeded code from `rasterizer/rasterizer.py`: including shrunk rasterization and all the abseil script stuff.
2. Makes the `rasterizer/` directory an importable module
3. Run a full raster upon validation.

TODO: fix the stop signal thing. What if a control point falls off-raster?

### CODE REVIEW

File: src/infer.py [hunk 1]
```diff
@@ -47,7 +48,7 @@ def infer(gen, num_fonts, path, style_dim=100, resolution=64, device='cuda'):
     # Generate a batch of fake characters
     fake_chars = gen(char_vector, style_vector)
     fake_chars = torch.reshape(fake_chars, [num_fonts, -1, 2])
-    rasters = rasterize(fake_chars)
+    rasters = rasterizer.Rasterizer()(fake_chars)
```
Comment (ccurro) [inline src/infer.py]: you don't use the resolution argument to the infer func. intentional?

Comment (eigenfoo) [reply src/infer.py]: No... not done with this PR yet actually.

Comment (ccurro) [reply src/infer.py]: Yeah I noticed the draft.

File: src/infer.py [hunk 2]
```diff
@@ -48,8 +50,15 @@ def infer(gen, num_fonts, path, style_dim=100, resolution=64, device='cuda'):
 
         # Generate a batch of fake characters
         fake_chars = gen(char_vector, style_vector)
-        fake_chars = torch.reshape(fake_chars, [1, -1, 2]).detach()
-        rasters = rasterize(fake_chars).detach()
+        fake_chars = torch.reshape(fake_chars, [len(CHARACTERS), -1, 3, 2]).detach()
+
+        # Rasterize glyph by glyph to avoid OOM. This _kills_ performance.
```
Comment (eigenfoo) [inline src/infer.py]: @ccurro I'm unhappy with this solution to OOMs. Any thoughts?

File: src/infer.py [hunk 3]
```diff
@@ -48,8 +50,16 @@ def infer(gen, num_fonts, path, style_dim=100, resolution=64, device='cuda'):
 
         # Generate a batch of fake characters
         fake_chars = gen(char_vector, style_vector)
```
Comment (ccurro) [inline src/infer.py]: what about requires_grad on this?

**Pull request merged**